mesheryctl/internal/cli/root/system/start.go

```diff
@@ -350,9 +349,32 @@ func start() error {
 				manifestFiles := filepath.Join(utils.MesheryFolder, utils.ManifestsFolder)
 
 				// change version in meshery-deployment manifest
-				err = utils.ChangeManifestVersion(utils.MesheryDeployment, channel, version, filepath.Join(manifestFiles, utils.MesheryDeployment))
-				if err != nil {
-					return err
+				if currCtx.GetVersion() == "latest" && currCtx.GetChannel() == "stable" {
```

Comment:
How about setting a channel=stable, version=latest in this case? and rest of the code will be common?

Reply:
agreed @hexxdump 
refactored in the latest commit.